bedrock/base/templates/base-protocol-basic.html

```diff
@@ -1,29 +1,31 @@
-{% from "macros.html" import side_nav_lists with context %}
+{% from "macros-protocol.html" import sidemenu_lists with context %}
 
-{% extends 'base-pebbles.html' %}
+{% extends 'base-protocol.html' %}
 
 {% block site_css %}
   {{ super() }}
-  {{ css_bundle('pebbles-basic') }}
+  {{ css_bundle('protocol-basic') }}
 {% endblock %}
 
 {% block body_id %}{{ body_id }}{% endblock %}
 
 {% block content %}
   {% block main_feature %}{% endblock %}
-  <main id="main-content" class="basic-content content">
-    <div class="basic-main{% if self.sidebar() %} basic-main-with-side{% endif%}">
-      {% block article %}{% endblock %}
-    </div>
+  <div id="main-content" class="mzp-l-content{% if self.side_nav()|trim|length or self.side_extra()|trim|length %} mzp-has-sidebar mzp-l-sidebar-right{% endif %}">
```

Comment:
This may well just be my brain, and I understand you're following how the old pages we're laid out, but there's something that just doesn't sit quite right with the navigation on the right, since it creates a (seemingly) arbitrary left padding on the main page content that's not aligned to the navigation:

![image](https://user-images.githubusercontent.com/400117/60732400-db277e80-9f41-11e9-8a20-dcddadd71b60.png)

Things feel more balanced to me when the menu is on the left?

![image](https://user-images.githubusercontent.com/400117/60732354-b4694800-9f41-11e9-8aa9-f16518f5c96f.png)

Reply:
It also currently results with the navigation coming last, after the main content on small screens.

I'm pretty sure there's also some JavaScript needed in order to make the side-menu functional on mobile, which currently seems to be missing.

Reply:
I think we should consider these two things separately.

1) Should the navigation come first in source order?

I put it after the content intentionally so people get to the content faster. I *think* this is a better experience for both mobile users and accessible tech users but I'm not sure and I'm not sure how to confirm it. 

With the menu collapsing on mobile it would be okay to have it first, but the other content in the aside will still be between users and the content.

2) Should the navigation be on the left?

I agree it looks better on the left with the current design.

Reply:
Yeah, it's pretty hard to know if the page content is what people want, or if they are navigating trying to find said content. There are a lot of links and sub pages in this part of the site.

If getting people to content faster is a proponent behind this decision, then perhaps a skip link might help, should the menu come first in the source order?